sdk/js-cloud-server/src/cloudClient.ts

```diff
@@ -96,7 +108,7 @@ export class DevCycleCloudClient {
 
         return getVariable(populatedUser, this.sdkKey, key, this.options)
             .then(async (res: Response) => {
-                const variableResponse = await res.json()
+                const variableResponse: any = await res.json()
```

Comment:
why any?

Reply:
I think due to the TS changes, it was treating these as `unknown`, which broke all these typings.